index.js

```diff
@@ -1,6 +1,8 @@
 //https://github.com/angular/angular.js/pull/10732
 
-var angular = require('angular');
+if (typeof angular !== 'undefined') {
```

Comment:
With variable hoisting the declaration of angular will be moved to the top of the file, which makes declare angular variable and set it to undefined. So the above condition fails, you need to have a different way to check whether angular is loaded or not